swift/swift.py

```diff
@@ -28,22 +28,31 @@ class Swift(QObject):
     Note that QApplication instance must be created before instantiating Swift object.
 
     Brief procedure:
-        1. Create buses.
-        2. Create apps.
-        3. Show frames of each app.
+        1. Load setup environment.
+        2. Create buses.
+        3. Create apps and show their frames.
+
+    Attributes:
+        _buses: A dictionary for storing buses.
+          Each element represents a bus.
+          A key is a bus name and its value is a Bus object.
+        _apps: A dictionary for storing apps.
+          Each element represents an app.
+          A key is an app name and its value is an App object.
+        _subscribers: A dictionary for storing subscribers.
+          Each element represents subscribers of each bus.
+          A key is a bus name and its value is a set containing its subscribers.

```

Comment:
We don't need to make docstrings for the internal attributes (whose names starting with _).
If you need to clarify some significant assumptions to those internal attributes, please use inline comments.
The docstring is for public API.

Reply:
Ok! I remove it.